### PR TITLE
profile icon moved to appbar and minor errors corrected

### PR DESCRIPTION
**Pull request by asare-21**
This commit fixes issue #98.

I also realised that room.tags was empty but was being accessed with [0]. I fixed it with an if stateme

### CODE REVIEW

**Comment by asare-21** [comment]
I also added the discussions icon to the bottom navigation bar

**Comment by chandansgowda** [comment]
@asare-21 Can you please add a screen recording to demo the changes ?

**Comment by asare-21** [comment]
Okay. I will do that

**Comment by asare-21** [comment]
https://github.com/AOSSIE-Org/Resonate/assets/52238457/b04b447a-134d-4188-8dab-897b4d4e4efc

**Review by chandansgowda — CHANGES_REQUESTED** [review]
Greta job till now @asare-21 
Need some changes-
1. Put the profile icon to the right of the appbar
2. Refactor logout button to profile screen (not required in app bar)
3. The profile icon should have the image of the user
4. Profile screen should have a title in the appbar

**Comment by asare-21** [comment]
Done refactoring. 


https://github.com/AOSSIE-Org/Resonate/assets/52238457/8bb2571e-9ba3-458a-9136-c6b49a8ca1c5

File: lib/views/widgets/profile_avatar.dart [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
-    ));
-}
\ No newline at end of file
+Widget profileAvatar(BuildContext context) {
+  return GestureDetector(
+    onTap: () => Navigator.pushNamed(context, AppRoutes.profile),
+    child: Padding(
+        padding: const EdgeInsets.only(right: 15),
+        child: GetBuilder<AuthStateController>(
+          builder: (controller) => CircleAvatar(
```
Comment (chandansgowda) [inline lib/views/widgets/profile_avatar.dart:12]: Please reduce the size of the circle avatar a bit and add a tiny amber border, explore ways to make it look even more better.

Comment (asare-21) [reply lib/views/widgets/profile_avatar.dart:12]: Noted

Comment (asare-21) [reply lib/views/widgets/profile_avatar.dart]: New changes pushed. Kindly have a look.

Comment (asare-21) [reply lib/views/widgets/profile_avatar.dart]: Simulator Screenshot - iPhone 14 Pro Max - 2023-08-02 at 06 10 12

Comment (chandansgowda) [reply lib/views/widgets/profile_avatar.dart]: Looks good. Can we reduce its size a little bit ?

**Comment by asare-21** [comment]
I've resized the icon and corrected the latest commit message

Simulator Screenshot - iPhone 14 Pro Max - 2023-08-02 at 22 09 29

**Comment by chandansgowda** [comment]
@asare-21 This looks good. Please resolve the merge conflicts.

**Comment by chandansgowda** [comment]
Hey @asare-21 
Please update me by EOD. This PR is blocking the other work that has to be done.

**Comment by asare-21** [comment]
okay

**Comment by asare-21** [comment]
Conflicts have been resolved. Sorry for the late reply

**Comment by asare-21** [comment]
@chandansgowda, I have resolved the conflicts

**Review by chandansgowda — APPROVED** [review]
Great job @asare-21 
@jddeep This PR can be merged.

**Pull request merged**